### PR TITLE
{chem}[intel/2021a] xTB v6.4.1

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/xTB/xTB-6.4.1-intel-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+version = '6.4.1'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = "Semiempirical Extended Tight-Binding Program Package."
+
+toolchain = {'name': 'intel', 'version': '2021a'}
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
```
Comment (branfosj) [inline easybuild/easyconfigs/x/xTB/xTB-6.4.1-intel-2021a.eb:15]: Please add the checksum.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/x/xTB/xTB-6.4.1-intel-2021a.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/x/xTB/xTB-6.4.1-intel-2021a.eb:15]: whoops, done

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/aec0ec9e0522049c19ef528259d6718a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/1c8544be093c1d4cd23db52cd0abb469 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**